src/components/List/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Dish from '../Dish';
+import Style from './styles.module.css';
+
+/**
+ * 
+ * @param {*} items - array of objects
+ * @param {*} listTitle - String
+ * @param {*} subTitle - String 
+ */
+const List = ({items, listTitle, subTitle}) => {
+    return (
+            <div className={Style.list}>
+                {(listTitle || subTitle) && <div className={Style.titles}>
+                {listTitle && <h2 className={Style.list_title}>{listTitle}</h2>}
+                {subTitle && <h3 className={Style.list_sub_title}>{subTitle}</h3>}
+            </div>}
+            <ul className={Style.ul}>
+                {items.map((item, index) =>{
+                    return (
+                    <>
```

Comment:
Avoid using fragment (<> </>) unnecessarily inside the map function. Instead, you can directly return the li element inside the map function.